library/src/scripts/theming/ThemeChooserInput.tsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+import { IComboBoxOption } from "@library/features/search/SearchBar";
+import { t } from "@vanilla/i18n";
+import React, { useEffect, useState } from "react";
+import { useThemeActions } from "@library/theming/ThemeActions";
+import { useThemeSettingsState } from "@library/theming/themeSettingsReducer";
+import { LoadStatus } from "@library/@types/api/core";
+import { DashboardSelect } from "@dashboard/forms/DashboardSelect";
+
+interface IProps {
+    value?: number | string | null;
+    initialValue: number | string;
+    onChange?: (value: number | string | null) => void;
+}
+
+export function ThemeChooserInput(props: IProps) {
```

Comment:
Could we get a docblock on this component? Just describe its purpose. The props don't need to be detailed because we have an interface for it.

Reply:
added